package.json

```
@@ -28,9 +28,11 @@
     "test:integration": "npm run start-test-wiremock && start-server-and-test start-feature http://localhost:3007/ping int-test",
     "test:integration:ui": "npm run start-test-wiremock && start-server-and-test start-feature:dev http://localhost:3007/ping int-test-ui",
     "test:e2e:ci": "npx playwright test --config ./e2e/playwright.config.ts --project=dev",
-    "test:e2e:local": "npx playwright test --config ./e2e/playwright.config.ts --project=local",
     "test:e2e:ui": "npm run test:e2e:ci -- --ui",
+    "test:e2e:local": "npx playwright test --config ./e2e/playwright.config.ts --project=local",
     "test:e2e:local:ui": "npm run test:e2e:local -- --ui",
+    "test:e2e:local-dev-upstream": "npx playwright test --config ./e2e/playwright.config.ts --project=local-dev-upstream",
```

If there was env-specific `.env` files, this could be something along those lines:

```suggestion
    "test:e2e:local-dev-upstream": "export $(cat .env.dev) && npx playwright test --config ./e2e/playwright.config.ts --project=local-dev-upstream",
```

---

that's great, thanks Fred. i'll have a play around with this

---

as discussed, we'll merge the PR as-is and pick this up in a subsequent PR